Sources/Commands/PackageTools/InstalledPackages.swift

```diff
@@ -0,0 +1,122 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2023-2022 Apple Inc. and the Swift project authors
```

Comment:
This year range is reversed. Does this reuse any code from previous years? If not, it could be just plain 2023.

```suggestion
// Copyright (c) 2023 Apple Inc. and the Swift project authors
```